tests/karma/unit/services/enketo-translation.js

```diff
@@ -731,7 +731,7 @@ describe('EnketoTranslation service', function() {
             '<instanceID/>' +
           '</meta>' +
         '</data>';
-      var element = $($.parseXML(model));
+      var element = $($.parseXML(model)).children().first();
```

Comment:
I added this because the [actual code does this](https://github.com/medic/medic-webapp/blob/2164_strict_binding/static/js/services/enketo-prepopulation-data.js#L18). AFAICT this only worked because our loose binding allowed it to.